### PR TITLE
Translate the social media section

### PR DESCRIPTION
**Pull request by nscyclone**
This refers to #453.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -138,3 +138,22 @@ toc:
...
+  vi:
+  ru:
+  sk:
+  fr:
+  gr:
+  it:
+  my:
+  pl:
+  no:
+  ko:
```
Comment (devleoper) [inline _config.yml]: In Korean, "이 페이지 공유하기" can be proper.

**Comment by nscyclone** [comment]
Thanks, @devleoper, I've just added your translation.

**Comment by nscyclone** [comment]
I've just mentioned some translators here. Sorry for bothering but I hope you don't mind to spend a minute to help us complete this.

File: _config.yml (hunk 1 above)
Comment (3100) [inline _config.yml]: In Japanese, It would be "このページをシェアする".

**Comment by nscyclone** [comment]
Thanks, @3100!

File: _config.yml (hunk 1 above)
Comment (therobinsj) [inline _config.yml]: In Norwegian, "Del denne siden"

**Comment by nscyclone** [comment]
Thanks, @therobinsj!

**Comment by doomspork** [comment]
Thank you for doing this @nscyclone!!

File: _config.yml (hunk 1 above)
Comment (michalvalasek) [inline _config.yml]: In Slovak: "Zdieľaj"

Comment (julienXX) [inline _config.yml]: In French: "Partager cette page".

**Comment by nscyclone** [comment]
Thanks, @michalvalasek and @julienXX!

File: _config.yml (hunk 1 above)
Comment (cizixs) [inline _config.yml]: In Chinese: 分享本页面

**Comment by nscyclone** [comment]
Thanks, @cizixs, I've just added your translation.

File: _includes/social.html [hunk 2]
```diff
@@ -5,7 +5,8 @@
   {% assign title = page.title %}
 {% endif %}
 <hr />
-<h3 id="social">Share This Page</h3>
+{% assign lang = page.lang || site.default_lang %}
```
Comment (devleoper) [inline _includes/social.html]: There is exactly the same code [at line 2](https://github.com/nscyclone/elixir-school/blob/18e45569ebb5397235bc3f7d50f0e525ad2ef1a7/_includes/social.html#L2).

How about moving this line at the top of this file, and remove the other line which does same work?

**Comment by nscyclone** [comment]
Thanks, @devleoper, I've just fixed that!

I've also moved the task list from this PR to the [corresponding issue](https://github.com/doomspork/elixir-school/issues/453). Probably we could merge this because there is almost no translation progress for a significant amount of time. What do you think about this, @doomspork ?

**Comment by doomspork** [comment]
I'm happy to merge this, should we use English for the missing translations?

**Comment by nscyclone** [comment]
@doomspork, yes, sure, English is used when no translation is provided in the config.

**Comment by doomspork** [comment]
Great!  This is good to merge, we can update the remainder as we go.

**Comment by christiansakai** [comment]
@nscyclone in ID, it is "Bagikan Halaman Ini"

File: _config.yml (hunk 1 above)
Comment (christiansakai) [inline _config.yml]: @nscyclone in ID, it is "Bagikan Halaman Ini"

Comment (nscyclone) [inline _config.yml]: Thanks, @christiansakai, I've just added it.

**Comment by nscyclone** [comment]
10/15 translations done. Thanks everyone!

**Pull request merged**